core/src/main/java/org/hibernate/ogm/loader/impl/OgmLoader.java

```diff
@@ -617,6 +620,12 @@ private TupleAsMapResultSet getResultSet(Serializable id, QueryParameters qp, Og
 		return resultset;
 	}
 
+	private TupleContext tupleContext(SessionImplementor session, OgmEntityPersister persister) {
+		TupleContext tupleContext = persister.getTupleContext( session );
+		tupleContext = new TupleContextImpl( tupleContext, session.getTransactionCoordinator() );
```

Comment:
This line seems redundant? The method 'persister.getTupleContext( session )' will already do the same, provided the seesion is not null. (and this method will NPE if it is, so I guess we expected it to be not null).